person.rb

```diff
@@ -1,16 +1,17 @@
 require './nameable'
 class Person < Nameable
+  attr_reader :rental, :id
```

Comment:
- [ ] Kindly add your rental variable to attribute accessor,  based on the requirements `Getters and Setters for @rentals should be present`

Reply:
Hello @prolajumokeoni thank you for your remarks I'll correct them right away

Reply:
Well resolved ✔️